pkg/mic/mic.go

```diff
@@ -175,6 +186,7 @@ func (c *Client) Sync(exit <-chan struct{}) {
 				if allBinding.Spec.Selector == crdPodLabelVal {
 					glog.V(5).Infof("Found binding match for pod %s/%s with binding %s", pod.Name, pod.Namespace, allBinding.Name)
 					matchedBindings = append(matchedBindings, allBinding)
+					nodeRefs[pod.Spec.NodeName] = true
```

Comment:
What happens in a case where the binding got deleted. Wouldn't we miss adding the node on which the binding was applied. Is that going to be an issue ?

Reply:
This happens before deletion so we are adding all nodes which have a binding at all.